packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.ts

```diff
@@ -327,6 +338,13 @@ export default ({
             )
             .getOrFail('Failed to get BTC receive address')
           break
+        case 'ETH':
+        case 'PAX':
+        case 'USDT':
+          receiveAddress = selectors.core.data.eth
+            .getDefaultAddress(yield select())
+            .getOrFail('Failed to get ETH receive address')
```

Comment:
for fail part do we always want to have ETH in it or maybe `Failed to get ${coin} receive address`?

Reply:
thanks! changed